src/vorta/qt_single_application.py

```diff
@@ -18,9 +17,6 @@ def __init__(self, id, *argv):
         super().__init__(*argv)
         self._id = id
 
-        if sys.platform.startswith('linux'):
```

Comment:
It does not seem to be necessary anymore, but I don't understand why.
Could please explain that?

Reply:
No idea. 🤷‍♂️